sklearn/random_projection.py

```diff
@@ -217,7 +217,7 @@ def _sparse_random_matrix(n_components, n_features, density='auto',
     n_features : int,
         Dimensionality of the original source space.
 
-    density : float in range ]0, 1] or 'auto', optional (default='auto')
+    density : float in range ]0, 1] or 'auto', default='auto'
```

Comment:
@amueller @adrinjalali @thomasjpfan does a non-French person understand this `]0, 1]` notation?

(just FYI the `(0, 1]` did look confusing to me at first)

Reply:
I understand either. In Germany the first is more common, but I'm more used to it on black board. I think sklearn is not very consistent with this.

Reply:
I may have seen `]0, 1]` but I'd definitely write `(0, 1]` myself

Reply:
Should I change it to `(0, 1]` ?

Reply:
yes please, I believe this is more internationally recognized

Reply:
@NicolasHug I made the necessary changes to it now